source/renderer/app/themes/overrides/OptionsOverrides.scss

```diff
@@ -1,11 +1,26 @@
 .options {
   margin-top: -7px;
 
-  ul {
-    border-radius: 4px;
-  }
-
   :global {
+    .SimpleOptions_hasSearch {
```

Comment:
@aleksandardjordjeviciohk I don't think you need to use :global here - hasSearch should be accessible as direct CSS class instead. Please consult with @DominikGuzei

Reply:
Added

Reply:
Can we please do the same for the `.SimpleOptions_search button:before {` 🙏

Reply:
Done